src/inline-svg.directive.ts

```diff
@@ -141,7 +141,7 @@ export class InlineSVGDirective implements OnInit, OnChanges, OnDestroy {
   private _insertEl(el: Element) {
     if (this.replaceContents && !this.prepend) {
       if (this._prevSVG) {
-        this._prevSVG.remove();
+        this._el.nativeElement.removeChild(this._prevSVG);
```

Comment:
I don't have access to a computer to work on actual code for the next couple days, so I can wait. :)